styles.css

```diff
@@ -0,0 +1,65 @@
+*{
+  box-sizing:border-box;
+}
+
+
+.u-list-reset {
+  list-style: none;
+  margin: 0;
+  padding-left: 0;
+}
+
+.u-list-inline li {
+  display: inline-block;
+  padding-left: 10px;
+  padding-left: 10px;
+}
+
+body {
+  background-color:hsl(234,18.5%,10.6%)
+}
+
+img {
+  float: left;
```

Comment:
Because you have used `float`, you must also clear your floats. See my docs on [using `float`](https://mica-web.github.io/learn/css/float) for more info on this.

If you did this, you could also eliminate the `height` rule on line 42 of your CSS.